README.md

```diff
@@ -98,6 +100,38 @@ async main() {
 }
 ```
 
+This example uses custom `socketTimeout` values and custom headers:
+
+```js
+const fetch = require('@adobe/node-fetch-retry');
+
+async main() {
+    const response = await fetch(url, {
+        retryOptions: {
+            retryMaxDuration: 300000, // 5min retry duration
+            socketTimeout: 60000, //  60s socket timeout
+        },
+        headers: {
+            'custom-header': '<<put custom header value here>>'
+        }
+    });
+}
+```
+
+This example uses only custom headers:
```

Comment:
Like above - this could give the impression `headers` are a custom feature of node-fetch-retry. Maybe move this example here to the top under "Default Behavior" as an example of fetch() with options and say that it supports normal options as well and will use the default retry settings.